src/main/java/com/laytonsmith/core/functions/StringHandling.java

```diff
@@ -1157,7 +1158,7 @@ public Construct exec(Target t, Environment environment, Construct... args) thro
 				params[i] = convertArgument(arg, c, i, t);
 			}
 			//Ok, done.
-			return new CString(String.format(formatString, params), t);
+			return new CString(String.format(Locale.US, formatString, params), t);
```

Comment:
Yes, but I do not like changing it to US locale, without giving them the
option to change it. It's fine as a default, but not as a forced choice.

On Fri, Sep 25, 2015 at 5:40 AM, Pietje <notifications@github.com> wrote:

> In src/main/java/com/laytonsmith/core/functions/StringHandling.java
> <https://github.com/EngineHub/CommandHelper/pull/341#discussion_r40417862>
> :
>
> > @@ -1157,7 +1158,7 @@ public Construct exec(Target t, Environment environment, Construct... args) thro
> >  				params[i] = convertArgument(arg, c, i, t);
> >  			}
> >  			//Ok, done.
> > -			return new CString(String.format(formatString, params), t);
> > +			return new CString(String.format(Locale.US, formatString, params), t);
>
> This fixes the failing unit test on all OS'es that use 1,23 as decimal
> seperator instead of 1.23.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EngineHub/CommandHelper/pull/341/files#r40417862>.
>